### DISCUSSION TITLE
[@types/random-words] Missing "seed" in the options

### DISCUSSION THREAD

**Discussion by Aryk** [DefinitelyTyped/DefinitelyTyped#67894] [Issues with a @types package] [answered]
The Options is missing `seed?: number | undefined`

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/random-words/index.d.ts

[a comment by an automated account is not shown]

**Comment by peterblazejewicz** [comment]
@Aryk I believe it's no longer relevant, please update your `random-words` to version 1.2.0 or newer , the package ships it's own types decoration afaik:
#67900

**Reply by Aryk** [reply, marked as answer]
Got it, thanks...we can close this issue
